sentry_sdk/integrations/asgi.py

```diff
@@ -147,6 +149,7 @@ async def _run_app(self, scope, callback):
                         transaction = Transaction(op="asgi.server")
 
                     transaction.name = _DEFAULT_TRANSACTION_NAME
+                    transaction.source = TRANSACTION_SOURCE_UNKNOWN
```

Comment:
so I don't think we should mark the default 'generic..' name as unknown because relay will then drop all those. The default name is low cardinality anyway since it's always the same value.

(following up in #client-infra)

Reply:
change to `route` as discussed on slack

Reply:
ok